lib/services/internalEngine/bootstrap.js

```diff
@@ -22,310 +22,148 @@
 'use strict';
 
 const
-  Bluebird = require('bluebird'),
   errorsManager = require('../../config/error-codes/throw'),
-  crypto = require('crypto');
+
 
 const
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Parse error at line 28 column 1:

18:  * See the License for the specific language governing permissions and
19:  * limitations under the License.
20:  */
21: 
22: 'use strict';
23: 
24: const
25:   errorsManager = require('../../config/error-codes/throw'),
26: 
27: 
28: const
    ^
29:   LOCK_ID = 'internalEngine.bootstrap.lock',
30:   JWT_SECRET_ID = 'security.jwt.secret',
31:   DATAMODEL_VERSION_ID = 'internalEngine.dataModelVersion';
32: 
33: class InternalEngineBootstrap {
34:   constructor (kuzzle) {
35:     this.kuzzle = kuzzle;
36:     this.storage = this.kuzzle.internalEngine;
37:   }
38: [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AParsingError)